spec.html

```diff
@@ -14189,6 +14206,53 @@ <h1>Runtime Semantics: Evaluation</h1>
         <emu-alg>
           1. Return GetNewTarget().
         </emu-alg>
+
+        <emu-grammar>ImportMeta : `import` `.` `meta`</emu-grammar>
+        <emu-alg>
+          1. Let _module_ be GetActiveScriptOrModule().
+          1. Assert: _module_ is a Source Text Module Record.
+          1. Let _importMeta_ be _module_.[[ImportMeta]].
+          1. If _importMeta_ is ~empty~, then
+            1. Set _importMeta_ to ! OrdinaryObjectCreate(*null*).
+            1. Let _importMetaValues_ be ! HostGetImportMetaProperties(_module_).
+            1. For each Record { [[Key]], [[Value]] } _p_ that is an element of _importMetaValues_,
+              1. Perform ! CreateDataProperty(_importMeta_, _p_.[[Key]], _p_.[[Value]]).
```

Comment:
```suggestion
              1. Perform ! CreateDataPropertyOrThrow(_importMeta_, _p_.[[Key]], _p_.[[Value]]).
```
the non-throw one returns false when it fails, so the ! doesn’t mean anything.